From bbf15c3ecaa93953cf3bfbb120fd4cfbeb0c8c70 Mon Sep 17 00:00:00 2001
From: Andrew Halaney <ahalaney@redhat.com>
Date: Tue, 18 Apr 2023 16:28:15 -0500
Subject: [PATCH 056/173] arm64: configs: laptop_defconfig: Update
 laptop_defconfig

Patchsets change, options change, and so do configs!
Before adding things to enable in the current tree, let's
update the defconfig to avoid churn in later patches:

[root@halaney-x13s linux]# make laptop_defconfig
[root@halaney-x13s linux]# make savedefconfig
[root@halaney-x13s linux]# cp defconfig arch/arm64/configs/laptop_defconfig
cp: overwrite 'arch/arm64/configs/laptop_defconfig'? y

Signed-off-by: Andrew Halaney <ahalaney@redhat.com>
---
 arch/arm64/configs/laptop_defconfig | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/arch/arm64/configs/laptop_defconfig b/arch/arm64/configs/laptop_defconfig
index 903ac5a6029d..56c9fa55a207 100644
--- a/arch/arm64/configs/laptop_defconfig
+++ b/arch/arm64/configs/laptop_defconfig
@@ -117,7 +117,6 @@ CONFIG_PARTITION_ADVANCED=y
 CONFIG_KARMA_PARTITION=y
 CONFIG_MQ_IOSCHED_KYBER=m
 CONFIG_IOSCHED_BFQ=m
-CONFIG_BFQ_GROUP_IOSCHED=y
 CONFIG_BINFMT_MISC=m
 CONFIG_ZSWAP=y
 CONFIG_Z3FOLD=m
@@ -377,7 +376,6 @@ CONFIG_IP_NF_TARGET_MASQUERADE=m
 CONFIG_IP_NF_TARGET_NETMAP=m
 CONFIG_IP_NF_TARGET_REDIRECT=m
 CONFIG_IP_NF_MANGLE=m
-CONFIG_IP_NF_TARGET_CLUSTERIP=m
 CONFIG_IP_NF_TARGET_ECN=m
 CONFIG_IP_NF_TARGET_TTL=m
 CONFIG_IP_NF_RAW=m
@@ -454,10 +452,8 @@ CONFIG_IEEE802154=m
 CONFIG_IEEE802154_6LOWPAN=m
 CONFIG_MAC802154=m
 CONFIG_NET_SCHED=y
-CONFIG_NET_SCH_CBQ=m
 CONFIG_NET_SCH_HTB=m
 CONFIG_NET_SCH_HFSC=m
-CONFIG_NET_SCH_ATM=m
 CONFIG_NET_SCH_PRIO=m
 CONFIG_NET_SCH_MULTIQ=m
 CONFIG_NET_SCH_RED=m
@@ -469,7 +465,6 @@ CONFIG_NET_SCH_CBS=m
 CONFIG_NET_SCH_ETF=m
 CONFIG_NET_SCH_TAPRIO=m
 CONFIG_NET_SCH_GRED=m
-CONFIG_NET_SCH_DSMARK=m
 CONFIG_NET_SCH_NETEM=m
 CONFIG_NET_SCH_DRR=m
 CONFIG_NET_SCH_MQPRIO=m
@@ -485,14 +480,11 @@ CONFIG_NET_SCH_PIE=m
 CONFIG_NET_SCH_INGRESS=m
 CONFIG_NET_SCH_PLUG=m
 CONFIG_NET_CLS_BASIC=m
-CONFIG_NET_CLS_TCINDEX=m
 CONFIG_NET_CLS_ROUTE4=m
 CONFIG_NET_CLS_FW=m
 CONFIG_NET_CLS_U32=m
 CONFIG_CLS_U32_PERF=y
 CONFIG_CLS_U32_MARK=y
-CONFIG_NET_CLS_RSVP=m
-CONFIG_NET_CLS_RSVP6=m
 CONFIG_NET_CLS_FLOW=m
 CONFIG_NET_CLS_CGROUP=m
 CONFIG_NET_CLS_BPF=m
@@ -643,7 +635,6 @@ CONFIG_ARM_SCMI_TRANSPORT_SMC_ATOMIC_ENABLE=y
 CONFIG_ARM_SCPI_PROTOCOL=y
 CONFIG_DMI_SYSFS=y
 CONFIG_FW_CFG_SYSFS=m
-CONFIG_QCOM_TEE_UEFISECAPP=y
 CONFIG_GOOGLE_FIRMWARE=y
 CONFIG_GOOGLE_COREBOOT_TABLE=m
 CONFIG_GOOGLE_MEMCONSOLE_COREBOOT=m
@@ -2364,7 +2355,6 @@ CONFIG_XEN_SCSI_BACKEND=m
 CONFIG_STAGING=y
 CONFIG_RTL8723BS=m
 CONFIG_R8712U=m
-CONFIG_R8188EU=m
 CONFIG_STAGING_MEDIA=y
 CONFIG_VIDEO_SUNXI=y
 CONFIG_VIDEO_SUNXI_CEDRUS=m
-- 
2.43.0

